### PR TITLE
Fix! Windows needs c: prefix for the token path

### PR DESCRIPTION
**Pull request by asincu**
## Description

Windows needs a `c:` prefix to access Linseed token path. For example, `c:/var/run/secrets/kubernetes.io/serviceaccount/token`

## For PR author

- [ ] Tests for change.
- [ ] If changing pkg/apis/, run `make gen-files`
- [ ] If changing versions, run `make gen-versions`

## For PR reviewers

A note for code reviewers - all pull requests must have the following:

- [ ] Milestone set according to targeted release.
- [ ] Appropriate labels:
  - `kind/bug` if this is a bugfix.
  - `kind/enhancement` if this is a a new feature.
  - `enterprise` if this PR applies to Calico Enterprise only.

### CODE REVIEW

**Review by matthewdupre — APPROVED** [review]

**Comment by rene-dekker** [comment]
Could you please reflect the change in fluentd_test at line 129?

**Review by rene-dekker — APPROVED** [review]
lgtm

**Comment by marvin-tigera** [comment]
OK, I will merge the pull request when it's ready, leave the commits as is when I merge it, and leave the branch after I've merged it.

**Pull request merged**
